packages/account/src/Components/api-token/__tests__/api-token-delete-button.spec.tsx

```diff
@@ -0,0 +1,95 @@
+import React from 'react';
+import { screen, render, waitFor } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { TApiContext, TToken } from 'Types';
+import ApiTokenContext from '../api-token-context';
+import ApiTokenDeleteButton from '../api-token-delete-button';
+
+const modal_root_el = document.createElement('div');
```

Comment:
Please move this assignments to `describe` suite and add cleanup functions to remove the mock modal element